core/tests/build_sources/templates/pages/Base.ts

```diff
@@ -18,9 +18,11 @@
 
```

Comment:
@nithusha21 could you review this file since, while I'm codeowner, it's not in a part of the code I have experience with? I'm blocking pending your review.

Reply:
Hi, the files in _build_sources_ are used for the testing ouf our build scripts. In general, it is good praxis to keep them updated with the rest of the codebase.

Reply:
I think the changes here are reasonable!